### PR TITLE
[MU4] Fix #328451: Add 2 (baroque) trills

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: https://musescore.org/en/node/328451
Applies to 3.x too

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Possible merge conflict with #10282

**Comment by Jojo-Schmitz** [comment]
rebase needed

**Comment by Jojo-Schmitz** [comment]
(rebase in progress)

**Comment by Jojo-Schmitz** [comment]
Any reason not to merge this?

File: src/engraving/libmscore/rendermidi.cpp [hunk 1]
```diff
@@ -1894,7 +1894,10 @@ std::vector<OrnamentExcursion> excursions = {
     { SymId::ornamentDownMordent,         any, _16th, { 1, 1, 1, 0 }, { 1, 0 },    true,  true, { -1, 0 } },// p136 Cadence Appuyee + mordent [1] [2]
     { SymId::ornamentPrallUp,             any, _16th, { 1, 0 }, { 1, 0 },        true,  true, { -1, 0 } },// p136 Double Cadence [1]
     { SymId::ornamentPrallDown,           any, _16th, { 1, 0 }, { 1, 0 },        true,  true, { -1, 0, 0, 0 } },// p144 ex 153 [1]
-    { SymId::ornamentPrecompSlide,        any, _32nd, {},    { 0 },          false, true, {} }
+    { SymId::ornamentPrecompSlide,        any, _32nd, {},    { 0 },          false, true, {} },
+
+    { SymId::ornamentShake3,              any, _32nd, { 1, 0 }, { 1, 0 },    true,  true, {} },
+    { SymId::ornamentShakeMuffat1,        any, _32nd, { 1, 0 }, { 1, 0 },    true,  true, {} },
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/rendermidi.cpp:1900]: For playback, these would now also need to be mentioned in /`src/engraving/playback/metaparsers/internal/symbolsmetaparser.cpp`.

File: src/engraving/types/symnames.cpp [hunk 2]
```diff
@@ -5296,8 +5296,8 @@ constexpr const std::array<const char*, size_t(SymId::lastSym) + 1> SymNames::s_
     "Right-facing hook",
     "Ornament right vertical stroke",
     "Schleifer (long mordent)",
-    "Shake",
-    "Shake (Muffat)",
+    QT_TRANSLATE_NOOP("symUserNames", "Shake"),
+    QT_TRANSLATE_NOOP("symUserNames", "Shake (Muffat)"),
```
Comment (cbjeukendrup) [inline src/engraving/types/symnames.cpp:5300]: Also `engraving/sym` here nowadays

File: src/engraving/libmscore/rendermidi.cpp (hunk 1 above)
Comment (cbjeukendrup) [reply src/engraving/libmscore/rendermidi.cpp:1900]: Uh, sorry. It is already 🤦

File: src/engraving/types/symnames.cpp (hunk 2 above)
Comment (Jojo-Schmitz) [reply src/engraving/types/symnames.cpp]: Done

**Comment by Jojo-Schmitz** [comment]
@RomanPudashkin mide to review and merge this one?

**Pull request merged**